input.example.yaml

```diff
@@ -32,7 +32,6 @@ cloudflare:
   # https://developers.cloudflare.com/workers/platform/routing/routes/
   routes:
     - https://YOUR_DOMAIN/*
-    - https://*.YOUR_DOMAIN/images/*
```

Comment:
Please also update the line numbers [here](https://github.com/google/sxg-rs/blob/3ac93c373acf6af24ecf2e11e6726e48db762b19/credentials/README.md?plain=1#L61).